### PR TITLE
Add support for methods 

### PR DESCRIPTION
**Pull request by gitcliff**

### CODE REVIEW

**Comment by gitcliff** [comment]
Screenshot from 2022-05-04 17-49-07
Screenshot from 2022-05-04 17-48-31

File: src/main/java/org/openelisglobal/method/daoimpl/MethodDAOImpl.java [hunk 1]
```diff
@@ -180,24 +181,39 @@ public MethodDAOImpl() {
...
-//
-//		} catch (RuntimeException e) {
-//			// bugzilla 2154
-//			LogEvent.logError("MethodDAOImpl", "getAllMethods()", e.toString());
-//			throw new LIMSRuntimeException("Error in Method getAllMethods()", e);
-//		}
-//
-//		return list;
-//	}
+	@Override
```
Comment (CalebSLane) [inline src/main/java/org/openelisglobal/method/daoimpl/MethodDAOImpl.java:184]: getAll and get (by Id) should already be implemented by the BaseDAOImpl that this class extends

Comment (CalebSLane) [reply src/main/java/org/openelisglobal/method/daoimpl/MethodDAOImpl.java:184]: the same should be true of BaseServiceImpl

File: src/main/java/org/openelisglobal/testconfiguration/controller/MethodRenameEntryController.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    public ModelAndView updateMethodRenameEntry(HttpServletRequest request,
+            @ModelAttribute("form") @Valid MethodRenameEntryForm form, BindingResult result) {
+        if (result.hasErrors()) {
+            saveErrors(result);
+            form.setMethodList(DisplayListService.getInstance().getList(DisplayListService.ListType.METHODS));
+            return findForward(FWD_FAIL_INSERT, form);
+        }
+
+        String methodId = form.getMethodId();
+        System.out.println("mmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm");
```
Comment (CalebSLane) [inline src/main/java/org/openelisglobal/testconfiguration/controller/MethodRenameEntryController.java:71]: System.out should be deleted

File: src/main/java/org/openelisglobal/renamemethod/daoimpl/RenameMethodDAOImpl.java [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+
+@Component
+@Transactional
+public class RenameMethodDAOImpl extends BaseDAOImpl<RenameMethod, String> implements RenameMethodDAO {
+
+    public RenameMethodDAOImpl() {
+        super(RenameMethod.class);
+    }
+   
+//	@Override
```
Comment (CalebSLane) [inline src/main/java/org/openelisglobal/renamemethod/daoimpl/RenameMethodDAOImpl.java:41]: Please delete these commented out methods in this new class

File: src/main/resources/liquibase/2.7.x.x/base.xml [hunk 4]
```diff
@@ -7,4 +7,7 @@
 
 	<include file="liquibase/2.7.x.x/address_line.xml"/>
 	<include file="liquibase/2.7.x.x/geographical_unit.xml"/>
+	<include file="liquibase/2.7.x.x/localization.xml"/>
```
Comment (CalebSLane) [inline src/main/resources/liquibase/2.7.x.x/base.xml:10]: @caseyi is this slated for 2.7 or should it be merged in to develop which is still at 2.6?

**Comment by caseyi** [comment]
Let’s merge it into 2.6 please. Cliff is just fast :D
________________________________

**Comment by gitcliff** [comment]
hello @CalebSLane kindly review

**Pull request merged**